.gitignore

```diff
@@ -28,9 +28,6 @@
 /public/500.html
 /public/503.html
 
-# Ignore vendor cache files
-/vendor
```

Comment:
I added `vendor/ruby` back in on another PR. For some reason my rbenv install really likes installing things into that directory. I'll try to review why it's doing that more later though.

Reply:
Oh I guess vendor/ruby is fine to have in there, just not vendor